spec/tcr_spec.rb

```diff
@@ -214,6 +222,17 @@
       end
     end
 
+    it "stubs out Socket#read correctly" do
+      TCR.configure { |c|
+        c.hook_tcp_ports = [23]
+      }
+      TCR.use_cassette("spec/fixtures/starwars_telnet") do
+        sock = TCPSocket.open("towel.blinkenlights.nl", 23)
+        expect(sock.read.length).to_not be_zero
```

Comment:
Well, the stubbing of the `IO#read` method is not entirely correct because a different `IO#read` with different arguments could return a wrong chunk.